schemas/context/person.schema.json

```diff
@@ -21,6 +21,12 @@
           "$ref": "https://ns.adobe.com/xdm/context/person-name",
           "description": "The person's full name"
         },
+        "xdm:birthDate": {
+          "title": "Birth Date",
+          "type": "string",
+	  "format": "date",
```

Comment:
As per the discussion, we don't want to have this to be `date` since the date might be incomplete. We discussed using a regular expression to match a date, so that we can be sure it is properly formatted.

Something along these lines:

```
"pattern": "([0-9X]{4}|XXXX)-([0-9]{2}|XX)-([0-9]{2}|XX)"
```

Reply:
I believe that pattern is an acceptable ISO format, is it not? If so, the date format is enough.

Reply:
It is @HyveMynd. The problem is how to deal with incomplete dates. For example, when we don't know the year.